src/Workspaces/Core/Portable/FindSymbols/FindReferences/Finders/IReferenceFinder.cs

```diff
@@ -29,22 +29,6 @@ internal interface IReferenceFinder
             FindReferencesSearchOptions options, FindReferencesCascadeDirection cascadeDirection,
             CancellationToken cancellationToken);
 
-        /// <summary>
-        /// Called by the find references search engine to determine which projects should be
-        /// searched for a given symbol.  The returned projects will then be searched in parallel. If
-        /// the implementation does not care about the provided symbol then null can be returned
-        /// from this method.
-        /// 
-        /// Implementations should endeavor to keep the list of returned projects as small as
-        /// possible to keep search time down to a minimum.  Returning the entire list of projects in
-        /// a solution is not recommended (unless, of course, there is reasonable reason to believe
-        /// there are references in every project).
-        /// 
-        /// Implementations of this method must be thread-safe.
-        /// </summary>
-        Task<ImmutableArray<Project>> DetermineProjectsToSearchAsync(
-            ISymbol symbol, Solution solution, IImmutableSet<Project>? projects, CancellationToken cancellationToken);
```

Comment:
there was exactly one implementation of this method, with no sublcasses overriding it.  So i just inlined the one impl at teh callsite.